api/all/src/main/java/io/opentelemetry/api/baggage/propagation/Element.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.api.baggage.propagation;
+
+import java.util.BitSet;
+
+class Element {
```

Comment:
even though this isn't public, it would be very helpful to have at least some description about what this class is about.

Reply:
Will do.